experimenter/experimenter/experiments/models.py

```diff
@@ -1130,6 +1130,14 @@ def get_changelogs_by_date(self):
 
         return transformed_changelogs
 
+    @property
+    def get_firefox_min_version_display(self):
```

Comment:
properties shouldnt begin with `get_` because you call them like attributes, not methods.

Reply:
This is specifically a [django naming convention](https://docs.djangoproject.com/en/5.0/ref/models/instances/#django.db.models.Model.get_FOO_display) for choice field display names